src/client.rs

```diff
@@ -600,6 +690,114 @@ impl Client {
     }
 }
 
+/// [Actions](https://www.w3.org/TR/webdriver1/#actions)
+impl Client {
+    /// Create a new Actions chain.
+    ///
+    /// ```ignore
+    /// let mouse_actions = MouseActions::new("mouse")
+    ///     .then(PointerAction::Down {
+    ///         button: MOUSE_BUTTON_LEFT,
+    ///     })
+    ///     .then(PointerAction::Move {
+    ///         duration: Some(Duration::from_secs(2)),
+    ///         origin: PointerOrigin::Pointer,
+    ///         x: 100,
+    ///         y: 0,
+    ///     })
+    ///     .then(PointerAction::Up {
+    ///         button: MOUSE_BUTTON_LEFT,
+    ///     });
+    /// client.actions(mouse_actions).perform().await?;
+    /// ```
+    ///
+    /// See the documentation for [`Actions`] for more information.
+    pub fn actions(&mut self, actions: impl Into<ActionSequence>) -> Actions {
```

Comment:
Ah, so, I proposed this split (`actions`/`perform`) only in relation to having a builder type. Since we abandoned that, I think this should now probably just be `perform_actions` :+1:

Reply:
@jonhoo `Actions` is currently now a builder type anyway. 

Do we want:
```rust
client.actions(first_actions).and(second_actions).perform().await?
```

or revert to the original

```rust
client.perform_actions(actions).await?;
```

I prefer the second one, in the spirit of not trying to make this too high-level

Reply:
I've changed it to the second option. Let me know if you prefer something else.

Reply:
Ah, I see what you mean. Hmm... In some sense I didn't think of `and` as a _really_ being a builder compared to what we had before. I could go either way on this one, so whichever you prefer. I think that _if_ you go the first route, `fn actions` should take no arguments and return an _empty_ action sequence, rather than requiring the user to pass in the first one.

Reply:
I think I'll stick with the second. Since the action chains could end up quite long and require multiple input sources etc, it feels better to separate the creation of the actions from the client method to perform them.